src/store/global-provider.tsx

```diff
@@ -1,22 +1,55 @@
 import React from 'react'
+import { GrowthBook, GrowthBookProvider } from '@growthbook/growthbook-react'
 import { BreakpointsProvider } from './breakpoints-context'
 import { PopupProvider } from './popup-context'
 import { RegionProvider } from './region-context'
 import { WebsiteStatusProvider } from './website-status-context'
 import { MediaContextProvider } from 'themes/media'
+import { getClientInformation, getDomain, getLanguage } from 'common/utility'
 
 type GlobalProviderProps = { children: React.ReactNode }
 
+let growthbook
+if (typeof window !== 'undefined') {
+    import('@deriv/analytics').then(({ RudderStack }) => {
+        RudderStack.init()
```

Comment:
We don't need to call `RudderStack.init()` anymore, because its automatically initialized once you import it

Reply:
<img width="240" alt="Screenshot 2023-08-01 at 4 47 49 PM" src="https://github.com/binary-com/deriv-com/assets/103016120/2fcce5e8-a09c-4cf5-ac60-2f90c0731aab">

Reply:
Great! Removed it now :)